types/nodegit/reference.d.ts

```diff
@@ -5,9 +5,9 @@ import { Repository } from "./repository";
 export namespace Reference {
     const enum TYPE {
         INVALID = 0,
-        OID = 1,
+        DIRECT = 1,
```

Comment:
```suggestion
        /** @deprecated Use {@link DIRECT} instead. */
        OID = 1,
        DIRECT = 1,
```

Reply:
added in [df31899](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/72430/commits/df318993918ec769a05fad8c2cb821df4ae66078)